src/components/Calculator.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+
+const keys = [
+  {
+    name: 'AC',
+    class: 'btn gray',
+  },
+  {
+    name: '+/-',
+    class: 'btn gray',
+  },
+  {
+    name: '%',
+    class: 'btn gray',
+  },
+  {
+    name: '÷',
+    class: 'btn orange',
+  },
+  {
+    name: '7',
+    class: 'btn gray',
+  },
+  {
+    name: '8',
+    class: 'btn gray',
+  },
+  {
+    name: '9',
+    class: 'btn gray',
+  },
+  {
+    name: 'x',
+    class: 'btn orange',
+  },
+  {
+    name: '4',
+    class: 'btn gray',
+  },
+  {
+    name: '5',
+    class: 'btn gray',
+  },
+  {
+    name: '6',
+    class: 'btn gray',
+  },
+  {
+    name: '-',
+    class: 'btn orange',
+  },
+  {
+    name: '1',
+    class: 'btn gray',
+  },
+  {
+    name: '2',
+    class: 'btn gray',
+  },
+  {
+    name: '3',
+    class: 'btn gray',
+  },
+  {
+    name: '+',
+    class: 'btn orange',
+  },
+  {
+    name: '0',
+    class: 'btn gray zero',
+  },
+  {
+    name: '.',
+    class: 'btn gray',
+  },
+  {
+    name: '=',
+    class: 'btn orange',
+  },
+];
+
+const Calculator = () => (
+  <div className="container">
+    <from>
+      <input type="text" placeholder="0" />
+    </from>
+    <div className="keypad">
+      {keys.map((btn) => (<button type="button" key={btn.name} className={btn.class}>{btn.name}</button>))}
+    </div>
+  </div>
+);
+
+export default Calculator;
```

Comment:
- [ ] Great job so far, just like you did in `App.js` kindly use class-based components, as it's one of the requirements.

![image](https://user-images.githubusercontent.com/59416136/170114838-578df49c-e126-4f5d-8dbb-c4706a502013.png)

You got this 👍.

Reply:
This issue is well resolved! ✔️